packages/react-liveness/src/components/FaceLivenessDetector/StartLiveness/helpers.ts

```diff
@@ -1,4 +1,4 @@
-export function getVideoConstraints(): MediaTrackConstraints | null {
+export function getVideoConstraints(): MediaTrackConstraints {
```

Comment:
Why does this need to be a function if the object it returns is static?

Reply:
Hm this is a leftover from before when we used to return different video constraints for mobile vs desktop. Should be safe to change it but wasn't sure if we should in this PR. What do you think?

Reply:
It would definitely be a bigger lift, but it does seem like we could simplify `videoConstraints` usage be making it a constant and referencing it directly where needed rather than passing around as a param between functions. Does that make sense?

Reply:
I'm good with changing it here

Reply:
Yep can make the change